emperor/support_files/js/color-editor.js

```diff
@@ -0,0 +1,133 @@
+/**
+ *
+ * @author Yoshiki Vazquez-Baeza
```

Comment:
I would suggest leaving this for another issue, lots of deletions,
perhaps worth it to leave as a future issue, as it is rather small and
mechanical.

On (Aug-20-15|15:20), Antonio Gonzalez wrote:
> > @@ -0,0 +1,133 @@
> > +/**
> > + *
> > + * @author Yoshiki Vazquez-Baeza
> 
> Should this be another issue? 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/407/files#r37587917